packages/components/src/components/link/shadow.tsx

```diff
@@ -117,7 +118,7 @@ export class KolLink implements LinkProps {
 	/**
 	 * Defines the description to use when the link is going to be opened in another application.
 	 */
-	@Prop() public _targetDescription?: string = 'Der Link wird in einem neuen Tab geöffnet.';
+	@Prop() public _targetDescription: string = translate('kol-open-link-in-tab');
```

Comment:
Das macht in der Doku nur Probleme. Eigentlich brauchen wir die Prop nicht. Daher nach Absprache mit Martin bitte entfernen.

Reply:
Ich bin über die Prop auch "gestolpert". Eine andere Option wäre noch, die Props durchgängig als optional ohne Initialisierung zu definieren und den Fallback-Wert erst bei der Übernahme im State zu haben?
@deleonio

Reply:
Ist jetzt entfernt, daher wäre die Frage hinfällig.